pkg/users/users.js

```diff
@@ -69,15 +69,19 @@ function AccountsPage() {
     const [max_uid, setMaxUid] = useState(60000);
     const [details, setDetails] = useState(null);
 
-    useInit(() => {
+    useInit(async () => {
+        const lastlogpath = await getLastlogPath();
         const debouncedGetLogins = debounce(100, () => {
-            getLogins().then(setDetails);
+            getLogins(lastlogpath).then(setDetails);
         });
 
-        // Watch `/var/run/utmp` to register when user logs in or out
-        const handleUtmp = cockpit.file("/var/run/utmp", { superuser: "try", binary: true });
-        handleUtmp.watch(() => debouncedGetLogins(), { read: false });
+        let handleUtmp;
```

Comment:
This could be renamed to `handleLastLog`.

Reply:
That is more appropriate will change